### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by hujw77**

### CODE REVIEW

File: contracts/lines/AxelarLine.sol [hunk 1]
```diff
@@ -9,9 +9,9 @@ import {AxelarExecutable} from "@axelar-network/axelar-gmp-sdk-solidity/contract
 import {IAxelarGateway} from "@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGateway.sol";
 import {IAxelarGasService} from "@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGasService.sol";
 import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/access/Ownable2Step.sol";
```
Comment (jiguantong) [inline contracts/lines/AxelarLine.sol:12]: Need in other lines?

Comment (hujw77) [reply contracts/lines/AxelarLine.sol:12]: Yes, other lines also use `Ownable2Step`.

Comment (jiguantong) [reply contracts/lines/AxelarLine.sol:12]: It seems that lzApp doesn't use Ownable2Step https://github.com/LayerZero-Labs/solidity-examples/blob/main/contracts/lzApp/LzApp.sol#L5

Comment (jiguantong) [reply contracts/lines/AxelarLine.sol:12]: And CelerLine

Comment (hujw77) [reply contracts/lines/AxelarLine.sol:12]: Yes, we use `Ownable2Step` include chain-id-mappings.
We could not change our upstream code except to implement Line in our way.

**Review by jiguantong — APPROVED** [review]

**Pull request merged**